packages/client-core/src/admin/components/Location/LocationTable.tsx

```diff
@@ -61,11 +63,9 @@ const LocationTable = ({ className, search }: Props) => {
   const openLocationDrawer = useHookstate(false)
   const locationAdmin = useHookstate<LocationType | undefined>(undefined)
 
-  const { t } = useTranslation()
-
   const adminLocations = useFind(locationPath, {
     query: {
-      $sort: {},
+      $sort: sortField.value ? { [sortField.value]: fieldOrder.value === 'desc' ? 0 : 1 } : {},
```

Comment:
Since, location service is based on feathers 5. Therefore, 0 is not valid anymore.

Kindly check the values here: https://feathersjs.com/api/databases/querying.html#sort

Reply:
I did not know that. There are a lot of changes which has 0 for descending. I shall fix them all.